frontend/src/GlobalStyles.js

```diff
@@ -4,16 +4,17 @@ export default createGlobalStyle`
 
 * {
     box-sizing: border-box;
-    font-family: "Open Sans", sans-serif;
+    font-family: 'Roboto', sans-serif;
```

Comment:
You load `Kanit` font in `index.html` and set the `font-family` to `Roboto`. Looks like there is a mistake :).

Reply:
`Kanit` is supposed to be used later for headings.

Reply:
In this case, you have to load Roboto too. Roboto is not installed on every OS by default.

Still, it's a good practice to install and add stuff only when you need it. I would add the fonts in the same branch when you use it.